src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    public Square(String color, double side) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Use local variables for better code readability.